app/backend/db/schema.sql.ts

```diff
@@ -72,6 +72,19 @@ export const sessionTableRelations = relations(sessionTable, ({ one }) => ({
 	}),
 }));
 
+export const profileResponseSchema = z.object({
+	// * Profile Informations to display
+
+	displayName: z.string().nullable(),
+	avatar_url: z.string().nullable(),
+
+	// * User Informations to display
+	username: z.string(),
+	email: z.string().email(),
+});
+
+export type ProfileResponse = z.infer<typeof profileResponseSchema>;
```

Comment:
The schemas for the profile are already created above using `drizzle-zod`. 

[more info](https://orm.drizzle.team/docs/zod#drizzle-zod)